en/integrations/plugin-packs/procedures/applications-pfsense-fauxapi.md

```diff
@@ -0,0 +1,179 @@
+| X         | APIKEY          | Pfsense Fauxapi key                                                        |
+| X         | APISECRET       | Pfsense Fauxapi secret                                                     |
+|    	    | APIEXTRAOPTIONS | Any extra option you may want to add to the command (eg. a --verbose flag) |
+
+## FAQ
+
+### How to check in the CLI that the configuration is OK and what are the main options for ?
+
+Once the Plugin installed, log into your poller using the *centreon-engine* user account and test by running the following command (Parameters such as ```api-key``` or ```api-secret```have to be adjusted):
+
+```bash
+/usr/lib/centreon/plugins/centreon_pfsense_fauxapi.pl \
+    --plugin=apps::pfsense::fauxapi::plugin \
+	--mode=gateways \
+	--hostname='api.pfsensefauxpi.com' \
+	--port='443' \
+	--proto='https' \
+	--api-key='myapikey' \
+	--api-secret='myapisecret' \
+	--filter-name='mygateway' \
+	--critical-status='%{status} !~ /none/i' \
+	--warning-packets-delay=120 \
+	--critical-packets-delay=300 \
+	--warning-packets-loss=5 \ 
+	--critical-packets-loss=10 \
+	--warning-packets-stddev=360 \
+	--critical-packets-stddev=480 \
+	--verbose
+
+OK: Gateways 'mygateway', Status: ok_mygateway |
+'gateway.packets.delay.milliseconds'=1;;120;300; 'gateway.packets.loss.percentage'=1;;5;10; 'gateway.packets.stddev.milliseconds'=0;;360;480;
+Gateways 'mygateway', Status: ok_mygateway
+```
+
+The command above gets the status of a gateway Pfsense Fauxapi (```--mode=gateways```) named *mygateway* (```--filter-name='mygateway'```). 
+It uses api-key (```--api-key='myapikey'```), an api-secret (```--api-secret='myapisecret'```)
+and it connects to the host _api.pfsensefauxpi.com_ (```--hostname='api.pfsensefauxpi.com'```) 
+on the port 443 (```--port='443'```) using https (```--proto='https'```).
+
+This command would trigger a CRITICAL alert if the returned status of the gateway is different from *none* (```--critical-status='%{status} !~ /none/i'```).
+
+This command will also trigger a WARNING alarm if the packets loss increase to 5% (```--warning-packets-loss='5'```)
+and a CRITICAL alarm if it increases to 10% (```--critical-packets-loss='10'```).
+Thresholds can be set on all of the device metrics using the syntax ```--warning-*metric* --critical-*metric*```.
+
+All the options that can be used with this plugin can be found over the ```--help``` command:
+
+```bash
+/usr/lib/centreon/plugins//centreon_pfsense_fauxapi.pl --plugin=apps::pfsense::fauxapi::plugin 
+--mode=gateways --help
+```
+
+### Why do I get the following error: 
+
+#### ```UNKNOWN: 500 Can't connect to api.pfsensefauxpi.com:443```
+
+This error message means that the Centreon Plugin couldn't successfully connect to the Pfsense Fauxapi.
+Check that no third party device (such as a firewall) is blocking the request.
+A proxy connection may also be necessary to connect to the API. This can be done by using the ```--proxyurl``` option in the command.
```

Comment:
```suggestion
A proxy connection may also be necessary to connect to the API. This can be done by using the ```--proxyurl='http://proxy.mycompany:8080'``` option in the command.
```